src/JK/WebsiteBundle/Controller/IndexController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace JK\WebsiteBundle\Controller;
+
+use JK\CommonBundle\Controller\AbstractController;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+
+/**
+ * @Route(service="jk.website.controller_index")
+ */
+class IndexController extends AbstractController
+{
+    /**
+     * @Route("/")
+     * @Template("JKWebsiteBundle:Default:index.html.twig")
```

Comment:
If you defined `template` service in abstract service controller, please use it here. Don't use `Template`